test.yml

```diff
@@ -10,18 +10,22 @@
     - name: 'Install software-properties-common'
       command: 'apt-get install -y software-properties-common'
       changed_when: false
+      when: ansible_distribution == 'Ubuntu'
```

Comment:
Quid of Debian ?

Reply:
Y a pas d'universe en debian, c'est purement ubuntu...